collections/landsat-4-5-tm-l1.yaml

```diff
@@ -291,50 +327,68 @@ Summaries:
           - 30
           - 30
         unit: m
+      data_type: float32
+      statistics:
+        minimum:
+          - 0
+        maximum:
+          - 0.4
     - description: Quality Assessment band (QA)
       name: BQA
       openeo:gsd:
         value:
           - 30
           - 30
         unit: m
+      data_type: UINT16
     - description: Radiometric Saturation and Terrain Occlusion QA Band
       name: QA_RADSAT
       openeo:gsd:
         value:
           - 30
           - 30
         unit: m
+      data_type: UINT16
     - description: View (sensor) Azimuth Angle
       name: VAA
       openeo:gsd:
         value:
           - 30
           - 30
         unit: m
+      data_type: INT16
     - description: View (sensor) Zenith Angle
       name: VZA
       openeo:gsd:
         value:
           - 30
           - 30
         unit: m
+      data_type: INT16
     - description: Sun Azimuth Angle
       name: SAA
       openeo:gsd:
         value:
           - 30
           - 30
         unit: m
+      data_type: INT16
```

Comment:
For some of the angles we have set FLOAT32. I think we should check for the Landsat collections what SH returns (decimal numbers or not) and update this accordingly.

Reply:
Checked the SH returns and updated accordingly.